app/queries/base_query.rb

```diff
@@ -1,29 +1,34 @@
 # frozen_string_literal: true
 
 class BaseQuery < BaseService
-  PER_PAGE = 100
-
-  Pagination = Struct.new(:page, :limit, keyword_init: true) do
-    def initialize(page: 0, limit: PER_PAGE)
-      super
-    end
-  end
+  Pagination = Struct.new(:page, :limit, keyword_init: true)
```

Comment:
this `PER_PAGE` default was not use and all the time it was using kaminari per page default (25) as limit was provided with `nil` value.

Also, first page is `1` instead of `0`, kaminari also defaults `nil` as first page.

hence, this code wasn't needed.